bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/message/PowerMaxSendType.java

```diff
@@ -0,0 +1,153 @@
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_EVENTLOG"),
+    DL_ZONESTR(new byte[] { 0x3E, 0x00, 0x19, 0x00, 0x02, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_ZONESTR"),
+    DL_ZONECUSTOM(new byte[] { 0x3E, (byte) 0xA0, 0x1A, 0x50, 0x00, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_ZONECUSTOM"),
+    DL_MR_ZONENAMES(new byte[] { 0x3E, 0x60, 0x09, 0x40, 0x00, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_MR_ZONENAMES"),
+    DL_MR_PINCODES(new byte[] { 0x3E, (byte) 0x98, 0x0A, 0x60, 0x00, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_MR_PINCODES"),
+    DL_MR_SIRENS(new byte[] { 0x3E, (byte) 0xE2, (byte) 0xB6, 0x50, 0x00, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 },
+            null, PowerMaxReceiveType.SETTINGS_ITEM, "DL_MR_SIRENS"),
+    DL_MR_KEYPADS(new byte[] { 0x3E, 0x32, (byte) 0xB7, 0x40, 0x01, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_MR_KEYPADS"),
+    DL_MR_ZONES(new byte[] { 0x3E, 0x72, (byte) 0xB8, (byte) 0x80, 0x02, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 },
+            null, PowerMaxReceiveType.SETTINGS_ITEM, "DL_MR_ZONES"),
+    DL_MR_SIRKEYZON(
+            new byte[] { 0x3E, (byte) 0xE2, (byte) 0xB6, 0x10, 0x04, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_MR_SIRKEYZON"),
+    DL_ALL(new byte[] { 0x3E, 0x00, 0x00, 0x00, (byte) 0xFF, (byte) 0xB0, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS_ITEM, "DL_ALL"),
+    SER_TYPE(new byte[] { 0x5A, 0x30, 0x04, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 }, null,
+            PowerMaxReceiveType.SETTINGS, "SER_TYPE"),
+    START(new byte[] { 0x0A }, null, PowerMaxReceiveType.SETTINGS, "START"),
+    EXIT(new byte[] { 0x0F }, null, null, "EXIT"),
+    POWERMASTER_ZONE_STAT1(
+            new byte[] { (byte) 0xB0, 0x01, 0x04, 0x06, 0x02, (byte) 0xFF, 0x08, 0x03, 0x00, 0x00, 0x43 }, null,
+            PowerMaxReceiveType.POWERMASTER, "POWERMASTER_ZONE_STAT1"),
+    POWERMASTER_ZONE_STAT2(
+            new byte[] { (byte) 0xB0, 0x01, 0x07, 0x06, 0x02, (byte) 0xFF, 0x08, 0x03, 0x00, 0x00, 0x43 }, null,
+            PowerMaxReceiveType.POWERMASTER, "POWERMASTER_ZONE_STAT2"),
+    POWERMASTER_ZONE_NAME(new byte[] { (byte) 0xB0, 0x01, 0x21, 0x02, 0x05, 0x00, 0x43 }, null,
+            PowerMaxReceiveType.POWERMASTER, "POWERMASTER_ZONE_NAME"),
+    POWERMASTER_ZONE_TYPE1(new byte[] { (byte) 0xB0, 0x01, 0x2D, 0x02, 0x05, 0x00, 0x43 }, null,
+            PowerMaxReceiveType.POWERMASTER, "POWERMASTER_ZONE_TYPE1");
+
+    private byte[] message;
+    private Integer paramPosition;
+    private PowerMaxReceiveType expectedResponseType;
+    private String description;
```

Comment:
Side fact: every `enum` has a `name()` method, which is the string version of the enum, so the `private String description` member seems to be redundant.